libraries/AP_HAL_ChibiOS/hwdef/BlitzF7/README.md

```diff
@@ -0,0 +1,89 @@
+# iFlight BLITZ Whoop F7 AIO Flight Controller
+
+https://shop.iflight-rc.com/BLITZ-Whoop-F7-AIO-Pro1927
+
+The Blitz F7 AIO is a flight controller produced by [iFlight](https://shop.iflight-rc.com/).
+
+## Features
+
+ - MCU: BGA-STM32F745, 216MHz
+ - Gyro: DPS310
+ - 16Mb Onboard Flash
+ - BEC output: 5V 2.5A
+ - Barometer: DPS310 or None
+ - OSD: AT7456E
+ - 5 UARTS: (UART1, UART2, UART3, UART4, UART7)
+ - I2C for external compass.
+ - 5 PWM outputs (4 motors and 1 LED)
+
+## Pinout
+
+![BLITZ Whoop F7 AIO Board](blitz_f7_pinout.jpg "BLITZ Whoop F7 AIO")
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+|Name|Pin|Function|
+|:-|:-|:-|
+|SERIAL0|COMPUTER|USB|
+|SERIAL1|RX1/TX1|UART1 (DJI connector, DMA-enabled)|
+|SERIAL2|TX2/RX2|UART2 (DJI connector RX, DMA-enabled, TX is on the back side of board)|
+|SERIAL3|TX3/RX3|UART3 (Alt RX, DMA-enabled)|
+|SERIAL4|TX4/RX4|UART4|
+|SERIAL7|TX7/RX7|UART7 (GPS)|
```

Comment:
this will not work by default...you would need to change protocol on Serial 3 and 4 and set the default for Serial 7 in hwdef to GPS protocol

Reply:
Thanks fixed.